azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/historypage.vm

```diff
@@ -251,6 +251,8 @@
                     <option value=90>Skipped</option>
                     <option value=100>Disabled</option>
                     <option value=110>Queued</option>
+                    <option value=120>Failed, treated as success</option>
```

Comment:
Is the message too long to be displayed ?

Reply:
I added screenshots in the description, please check it out. I don't think it's too bad. I think this label makes it clear to the users. Can you think of something else that would be as clear as this, without being clumsy? It's a special case after all, won't be too common to appear in practice.

Reply:
I didn't check out the screenshot. Looks good. +1 on the long message for detailed explanation. Otherwise, users should not understand what does FAILED_SUCCEEDED mean ? I always get confused, too.